src/test/java/bio/overture/ego/utils/EntityGenerator.java

```diff
@@ -269,6 +284,8 @@ public User generateRandomUser() {
             .preferredLanguage(randomLanguageType())
             .firstName(randomStringNoSpaces(5))
             .lastName(randomStringNoSpaces(6))
+            .identityProvider(GOOGLE)
```

Comment:
create a constant called `DEFAULT_PROVIDER_TYPE = GOOGLE`, and then reference that.